backend/census_historical_migration/sac_general_lib/audit_information.py

```diff
@@ -0,0 +1,150 @@
+from census_historical_migration.workbooklib.census_models.census import (
+    CensusGen22 as Gen,
+    CensusCfda22 as Cfda,
+    CensusFindings22 as Finding,
+)
+from census_historical_migration.base_field_maps import FormFieldMap, FormFieldInDissem
+from census_historical_migration.sac_general_lib.utils import (
+    _create_json_from_db_object,
+)
+import audit.validators
+
+DOLLAR_THRESHOLD = 750000
+
+mappings = [
+    FormFieldMap(
+        "dollar_threshold", None, FormFieldInDissem, DOLLAR_THRESHOLD, int
+    ),  # FIXME: There is no in_db mapping ?
+    FormFieldMap(
+        "gaap_results", None, FormFieldInDissem, [], list
+    ),  # FIXME: There is no in_db mapping?
+    FormFieldMap(
+        "is_going_concern_included", "goingconcern", FormFieldInDissem, None, bool
+    ),
+    FormFieldMap(
+        "is_internal_control_deficiency_disclosed",
+        "materialweakness",
+        FormFieldInDissem,
+        None,
+        bool,
+    ),
+    FormFieldMap(
+        "is_internal_control_material_weakness_disclosed",
+        "materialweakness_mp",
+        FormFieldInDissem,
+        None,
+        bool,
+    ),
+    FormFieldMap(
+        "is_material_noncompliance_disclosed",
+        "materialnoncompliance",
+        FormFieldInDissem,
+        None,
+        bool,
+    ),
+    FormFieldMap(
+        "is_aicpa_audit_guide_included",
+        "reportablecondition",
+        FormFieldInDissem,
+        None,
+        bool,
+    ),
+    FormFieldMap(
+        "is_low_risk_auditee", None, FormFieldInDissem, False, bool
+    ),  # FIXME: There is no in_db mapping?
+    FormFieldMap(
+        "agencies", None, "agencies_with_prior_findings", [], list
+    ),  # FIXME: There is no in_db mapping?
+]
+
+
+def _get_agency_prefixes(dbkey):
+    agencies = {}
+    cfdas = Cfda.select().where(Cfda.dbkey == dbkey)
+    cfda: Cfda
+    for cfda in cfdas:
+        agencies[int((cfda.cfda).split(".")[0])] = 1
+    return agencies.keys()
+
+
+def _get_gaap_results(dbkey):
+    findings = Finding.select().where(Finding.dbkey == dbkey)
+    gaap_results = {}
+    # FIXME: How do we retrieve gaap_results from the historic data? I could not find corresponding fields in Census tables.

```

Comment:
It has a different name. We might ask @Carley-Jackson .

Reply:
[Ticket ](https://github.com/GSA-TTS/FAC/issues/2832) created for follow up.